server.py

```diff
@@ -0,0 +1,39 @@
+from flask import Flask, request, make_response, json
+#from find_melanoma import Melanoma
+import base64
```

Comment:
If this is only used in `patient_prediction()`, then I would import there (it would make testing--not required for this project--to be done more easily).

Reply:
yes, I uncomment that part.